src/components/JobCard/JobCard.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styles from './JobCard.module.scss';
+import { IJob } from '../../services/types';
+
+function JobCard(props: IJob) {
+   return (
+      <div className={styles['offer']}>
```

Comment:
**nitpick:** It's easier to navigate when using `styles.offer` instead of using string literals. This naming convention is up to you, but it should be consistent everywhere - so, `styles.className` or `styles['className']`